R/measures.R

```diff
@@ -105,7 +105,7 @@ riem_measures <- function(station = "VOHY",
     return(NULL)
   }
 
-  result$valid <- lubridate::ymd_hm(result$valid)
+  result$valid <- lubridate::as_datetime(result$valid,format="%F %H:%M")
```

Comment:
do you think we could remove this change since it's a lubridate bug fixed in its dev version?